includes/functions/functions_dates.php

```diff
@@ -196,7 +196,7 @@ function zen_checkdate($date_to_check, $format_string, &$date_array)
         }
     }
 
-    if ($date_separator_idx != $format_separator_idx) {
+    if (!isset($date_separator_idx) || !isset($format_separator_idx) || $date_separator_idx != $format_separator_idx) {
```

Comment:
Please combine those two `!isset()`'s into `!isset($date_separator_idx, $format_separator_idx)`.  If either variable is 'not set', that condition will be  `true`.